plugins/gatsby-plugin-apollo/client.js

```diff
@@ -7,7 +7,9 @@ export const client = new ApolloClient({
     const isBrowser = typeof window !== 'undefined'
     const token = isBrowser ? localStorage.getItem('token') : null
     operation.setContext({
-      uri: 'https://recipe-palette.herokuapp.com/v1/graphql',
+      uri:
+        process.env.GATSBY_HASURA_URL ||
+        'https://recipe-palette-dev.herokuapp.com/v1/graphql',
       headers: {
         'content-type': 'application/json',
         'x-hasura-admin-secret': 'Something 2 think about!',
```

Comment:
Just realized we should probably take out or admin-secret.. 😅

Reply:
Hahahah yeah that's probably true, we can move it into an env file, should probably change it or something to remove it from the git history as well